features/compose_message.feature

```diff
@@ -0,0 +1,21 @@
+Feature: Compose messages
+  As a user
+  In order to send a messeage
+  I should be able to compose a message
+
+  Background:
+    Given the following user exist
```

Comment:
well if we're creating more than 1 user then we should say `following users exist:`